FluentUI.Demo/src/main/res/values/styles.xml

```diff
@@ -6,10 +6,112 @@
         <item name="windowNoTitle">true</item>
-        <item name="android:windowBackground">?attr/fluentuiBackgroundColor</item>
+        <item name="homeAsUpIndicator">@drawable/ms_ic_arrow_left_24_filled_toolbar</item>
+        <item name="android:windowBackground">@color/fluentui_white</item>
+        <!--Launch Screen: Remove the background that was set in the launch screen theme-->
+        <item name="android:background">@null</item>
+        <item name="android:statusBarColor">?android:colorPrimary</item>
+        <item name="android:windowLightStatusBar" tools:targetApi="23">false</item>
+
+
+        <!-- If you decide to use any of the attributes defined in Fluent Library in the demo app ,
+             then those attrs should be defined in your theme or they should be wrapped in
+             FluentUIContextWrapper if using programmatically -->
+        <item name="fluentuiBackgroundColor">@color/fluentui_white</item>
+        <item name="fluentuiForegroundColor">@color/fluentui_gray_900</item>
+        <item name="fluentuiForegroundSecondaryColor">@color/fluentui_gray_500</item>
+
+
+        <!-- Some of the Styles like Toolbar, Button, BottomNavigation or any other style supported
+             by android can be directly set in the Theme. These can be used in 3 ways -
+
+             1) Define your own custom style - Recommended for cases when you are not using the
+                    components provided by Fluent library. for example if you want to use your own
+                    buttons throughout the app and not FluentUI.Widget.Button. Also, if you decide
+                    to use Fluent UI Components then you can set the style directly in XML file.
+             2) Use Style Defined in Library - Recommended for cases when you use Fluent UI
+                     component throughout the app. It will directly pick the attrs defined in the
+                     styles and hence you do not need to declare them. But if in case you want some
+                     attrs to be overridden you can easily do by re-declaring them. For example
+                     <item name="fluentuiToolbarTitleTextColor">#ff0000</item> will change the
+                     toolbar title color to red given you have set the style
+             3) Use Style Defined in Library but using your own Widget - Not Recommended. But still
+                    if you decide to do this.
+                    Example - You have set Fluent Library Style
+                    <item name="buttonStyle">@style/Widget.FluentUI.Button</item>
+                    But in app you are not using FluentUI.Widget.Button and directly consuming
+                    Button provided by android. So, you will face inconsistencies in color and
+                    other attributes. To not face inconsistencies you will have to declare the
+                    attrs defined in Library again.
+                    <item name="fluentuiButtonBackgroundDefaultColor">your_color</item>
```

Comment:
Question : What's the way to know all available attribute's name I want to override  in 2nd & 3rd case ?